### ISSUE TITLE
session never works if login filled with email

### ISSUE THREAD

**Issue by AndCycle** [instaloader/instaloader#624]
**Describe the bug**
the session file never works if I filled username with my login email,
always get a new login warning from Instagram which will trigger ip block really soon.

**To Reproduce**
```instaloader +insta_args.txt :feed :stories --fast-update --commit-mode```

insta_args.txt
```
--login=mylogin@email.address
--password=mylogin_password
--sessionfile=/home/insta/login-session.txt
--dirname-pattern=store/{profile}
```

**Instaloader version**
4.4.2

**Additional context**
not an actual bug report as I found this little detail,
I used to login with my email address,
it would be nice if there can be some warning about this.

**Comment by evergreenschool** [comment]
I have the same issue with username login.

**Comment by mkturner** [comment]
Is that why I'm getting this:
`Fatal error: Login error: JSON decode fail, 400 - Bad Request.`

**Comment by AndCycle** [comment]
@mkturner 
your issue been fixed in 4.4.2 https://github.com/instaloader/instaloader/issues/620

**Comment by aandergr** [comment]
The `--login` flag is thought to be used with the username. Passing the email address to `--login` is not supported.
